src/workers/postUpdated.ts

```diff
@@ -291,6 +327,7 @@ const fixData = async ({
   // Try and fix generic data here
   return {
     mergedKeywords,
+    content_type: data?.content_type as PostType,
```

Comment:
In this flow do you need `content_type` at all?

Reply:
Right now only the updatePost uses `content_type`, but I added it to fixData in case createPost would need it in the future.
I can remove it from fixData if that is preferred.

Reply:
Ah i see, no that's fine with me.
Just didn't see any use so wasn't sure.

Let's keep it for now :)